.github/workflows/inspektor-gadget.yml

```
@@ -1,6 +1,7 @@
 name: Inspektor Gadget CI
 env:
-  CONTAINER_REPO: ${{ secrets.CONTAINER_REPO }}
+  REGISTRY: ghcr.io
+  CONTAINER_IMAGE: ${{ (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v')) && format('{0}', github.repository) || format('{0}-ci', github.repository) }}
```

Maybe using additional parenthesis makes it easier to read?

```suggestion
  CONTAINER_IMAGE: ${{ ((github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v')) && format('{0}', github.repository)) || format('{0}-ci', github.repository) }}
```

---

I added them but I am not sure it makes the thing easier to read.
The best would be to have this split on several lines but I am not sure this is syntatically valid.